app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewModel.kt

```diff
@@ -135,18 +139,42 @@ class AttendeeViewModel(private val attendeeService: AttendeeService, private va
                 }.doFinally {
                     progress.value = false
                 }.subscribe({
-                    if (attendee.ticket?.id != null) {
-                        loadTicket(attendee.ticket?.id, country, eventId, paymentOption, it.id)
+                    attendees.value?.add(it)
+                    if (attendees.value != null && attendees.value?.size == totalAttendee) {
+                        loadTicketsAndCreateOrder()
                     }
                     message.value = "Attendee created successfully!"
-                    Timber.d("Success! %s", it.id)
+                    Timber.d("Success! %s", attendees.value?.toList().toString())
                 }, {
                     message.value = "Unable to create Attendee!"
                     Timber.d(it, "Failed")
                 }))
     }
 
-    fun loadTicket(ticketId: Long?, country: String, eventId: Long, paymentOption: String, attendeeId: Long) {
+    fun createAttendees(attendees: List<Attendee>, country: String, paymentOption: String) {
+        if (this.attendees.value == null) {
+            this.attendees.value = ArrayList()
+        }
+        this.country = country
+        this.paymentOption = paymentOption
+        this.attendees.value?.clear()
```

Comment:
Why?

Reply:
so that everytime this function is called prev attendee data is cleared ( `createAttendees` calls `createAttendee` for every attendee to be created) same applies to ticket